stylesheets/components/badges.css

```diff
@@ -0,0 +1,91 @@
+.badge-wrapper {
+  background-color: var(--baltic-sea-color);
+  border: 1px solid var(--charcoal-grey);
+  border-radius: var(--border-rd-6);
+  display: flex;
+  min-width: 16rem;
+  justify-content: space-around;
+  padding: 1rem 1.5rem;
+}
+.badge {
+  position: relative;
+}
+.badge-round{
+    position: absolute;
+    z-index: 1;
+    border-radius: 50%;
+}
+/* icons with badge */
+.badge-icon {
+  color: var(--blue-semi-dark);
+  font-size: 2.5rem !important;
+  padding-top: 0.5rem;
+  padding-right: 0.4rem;
+  padding-left: 0.2rem;
+}
+.badge-counter-right {
+  top: 0;
+  right: 0;
+}
+.badge-counter-left {
+  top: 0;
+  left: 0;
+}
+
+.badge-counter { 
+
+  color: var(--white-color);
+  background-color: var(--red-light);
+  padding: 1px 0.4rem;
+}
+/* avatar badge */
+.avatar-badge {
+  width: clamp(2rem, 3.5rem, 3.5rem);
+}
+.avatar-img {
+  border-radius: 50%;
+}
+.badge-status { 
+  padding: 1px 0.4rem;
+}
+.badge-status-dot {
+  min-height: 13.5px;
+  background-color: var(--forest-green);
+  right: 0;
+  bottom: 0.2rem;
+}
+.badge-status-circle {
+  top: 0;
+  height: 100%;
+  width: 100%;
+  border: 2.5px solid var(--blue-semi-dark);
+}
+/* text with badge */
+.badge-text-content {
+  padding-top: 0.8rem;
+  padding-right: 0.5rem;
+  color: var(--white-color);
+  font-size: 1.2rem;
+}
+
+.badge-counter-lg {
+  width: 1.9rem;
+  height: 1.9rem;
+  display: flex;
+  top: -6px;
+}
+/* buttons with badge */
+.badge-btn {
+  margin-top: 1rem;
+  margin-right: 0.5rem;
+  margin-left: 0.5rem;
+  padding: 0.5rem 1rem;
+  color: var(--white-color);
+  border-radius: 6px;
+}
+.badge-btn-primary {
```

Comment:
you can use button classes to avoid extra code.

Reply:
Thank you, Vivek for this suggestion! This really helped in minimizing the code